<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>79</width>
         <height>486</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_v4">
          <property name="text">
           <string>Initial
run</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="pushButton_v8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Weight
distribution</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_v1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Resonance
test</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="pushButton_v9">
          <property name="text">
           <string>Add
vectors</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_v7">
          <property name="text">
           <string>Save
rotor</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_v3">
          <property name="text">
           <string>Rotor
setup</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton_v6">
          <property name="text">
           <string>Trail
run 2
(testdialog3sub)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPushButton" name="pushButton_v11">
          <property name="text">
           <string>Tool
library</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="pushButton_v10">
          <property name="text">
           <string>Rotor
files</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_v5">
          <property name="text">
           <string>Trail
run 1
(testdialog)</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="pushButton_v12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_v2">
          <property name="text">
           <string>&quot;Spectrum&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_0_resonance">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Here we have resonance test &quot;stack-item&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_1_spectrum">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string>L</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0,000 mm/s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0 rpm</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0,000 mm/s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>823</width>
             <height>384</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>823</width>
             <height>69</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_h1">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Help</string>
              </property>
             </widget>
            </item>
            <item row="0" column="9">
             <widget class="QPushButton" name="pushButton_h10">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_h2">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Print</string>
              </property>
             </widget>
            </item>
            <item row="0" column="8">
             <widget class="QPushButton" name="pushButton_h9">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Measure</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_h3">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Static
Couple</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QPushButton" name="pushButton_h7">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Trace on</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton_h4">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Remove
wights</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="pushButton_h6">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Change
unit</string>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QPushButton" name="pushButton_h8">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="pushButton_h5">
              <property name="styleSheet">
               <string notr="true">margin: 2px; border-color: #009900; border-style: outset; border-radius: 20px; border-width: 4px; height:4em; width: 4em;</string>
              </property>
              <property name="text">
               <string>Split On</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2_rotor_setup">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Rotor setup view</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3_initial_run">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Initial run view</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4_test_run_1">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>This is shown when you press Trail run 1, but you should see this.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="6">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_h10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1447</x>
     <y>863</y>
    </hint>
    <hint type="destinationlabel">
     <x>1479</x>
     <y>903</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
